elasticsearch/src/main/java/com/tcdi/zombodb/highlight/AnalyzedField.java

```diff
@@ -5,9 +5,9 @@
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- *
+ * 
```

Comment:
This was was just a auto deletion of spaces in the end of lines which I turned off later. I' tried to restore the original. Sorry for that noise.